plugins/by-name/oil-git-status/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  config,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "oil-git-status";
+  packPathName = "oil-git-status.nvim";
+  package = "oil-git-status-nvim";
+
+  description = ''
+    Add Git Status to oil.nvim directory listings.
+
+    Git status is added to the listing asynchronously after creating the oil directory
+
+    listing so it won't slow oil down on big repositories. The plugin puts the status in two new sign columns
+
+    the left being the status of the index, the right being the status of the working directory
+
+    `win_options.signcolumn` needs to be configured in `plugins.oil.settings`:
+    ```nix
+    plugins.oil = {
+      enable = true;
+      settings = {
+        win_options = {
+          signcolumn = "yes:2,";
+        };
+      };
+    };
+    ```
+  '';
+
+  maintainers = [ lib.maintainers.FKouhai ];
+
+  # Ensure oil-git-status is set up after oil is loaded
+  configLocation = lib.mkOrder 1100 "extraConfigLua";
+
+  dependencies = [
+    "git"
+  ];
+
+  extraConfig = cfg: {
+    warnings = lib.nixvim.mkWarnings "plugins.oil-git-status" [
+      {
+        when = !config.plugins.oil.enable;
+        message = ''
+          This plugin requires `plugins.oil` to be enabled
+          plugins.oil.enable = true;
+        '';
+      }
+      {
+        when = !config.plugins.oil.settings.win_options.signcolumn or null != "yes:2,";
```

CI failure, you have an extra `!`
```suggestion
        when = config.plugins.oil.settings.win_options.signcolumn or null != "yes:2,";
```

https://buildbot.nix-community.org/#/builders/2876/builds/4532

---

oh man I missed that thanks :)